wintaser/Movie.cpp

```diff
@@ -244,17 +248,13 @@ Movie::Movie()
 
 	unsigned int commandlineLen;
 	fread(&commandlineLen, 4, 1, file);
-	char* commandline;
-	commandline = NULL;//= movie.commandline;
+	char* commandline = NULL;//= movie.commandline;
 	if(commandlineLen) // There's a command line in the movie file.
 	{
 		if(commandlineLen > (8192-1)-(MAX_PATH+1)) // We have exceeded the maximum allowed command line. Something's wrong.
 		{
 			fclose(file);
-			if(author) // Prevent memory leak
-			{
-				delete [] author;
-			}
+			delete [] author; // Prevent memory leak

```

Comment:
I would consider it unsafe to provide a delete that goes crazy with nullptr.... But well I can (re)add a nullptr check for every deallocation call in this function.